### DISCUSSION TITLE
Update a cask to a newer version with a lower version number

### DISCUSSION THREAD

**Discussion by WangEdward** [Homebrew/discussions#4553] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
-
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

The version for xmind at [latest branch](https://github.com/Homebrew/homebrew-cask/blob/4021dc08074e7eda18528bc4bf034bd5266c3693/Casks/xmind.rb) is `23.05.2661-202305210750`
However, on June 2nd it release a newer version with varsion number [`22.11.3771-202305311748`](https://web.archive.org/web/20230603031350/https://xmind.app/desktop/release-notes/)
```bash
➜ brew livecheck xmind
xmind: 23.05.2661-202305210750 ==> 22.11.3771-202305311748
```
Which version should we have in the https://github.com/Homebrew/homebrew-cask repo?
https://github.com/Homebrew/homebrew-cask/pull/148435

**Comment by gromgit** [comment, marked as answer]
Please check with upstream why they "backstepped" the version. The Git log for the cask shows there have already been **four** updates in the `23.05` series, so this might actually be an upstream release error.
